apps/shopify/src/skuResolvers.js

```diff
@@ -9,6 +9,12 @@ import { productDataTransformer, collectionDataTransformer } from './dataTransfo
 
 import { validateParameters } from '.';
 import { previewsToProductVariants } from './dataTransformer';
+import { DEFAULT_SHOPIFY_API_VERSION } from './constants';
+import isBase64 from './utils/isBase64';
+import btoa from './utils/btoa'
+import atob from './utils/atob'
```

Comment:
Maybe worth putting these base64-related utils to single module? wdyt?

Reply:
makes sense